snippets/csharp/VS_Snippets_CLR/conceptual.basicio.textfiles/cs/file.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.IO;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        // Create a string array with the lines of text
+        string text = "First line" + Environment.NewLine;
```

Comment:
@v-thepet comment doesn't match the code: not a string array is created.

Though, I wonder how much the comments are necessary: the code is self-documenting and additional lines make grasping it slower. Moreover, code comments are not localized.

Reply:
Thanks for noticing the incorrect comment, @pkulikov. I've updated the comment so that it's now accurate, @v-thepet , though I think that it's worthwhile to consider whether the comments are useful at all, especially since they aren't localized.  I'm going to merge this PR so that dotnet/docs#9760 can be built and reviewed. But in reviewing it, we should decide whether the comments should be removed and possibly incorporated into the descriptive text.